### DISCUSSION TITLE
截图中的参数是怎么使用的呢?

### DISCUSSION THREAD

**Discussion by SoyKoa** [shuax/MouseInc.Settings#442] [Q&A]
![image](https://user-images.githubusercontent.com/76244289/192196982-9eeb3ec7-fc7a-4f70-a926-35a3f59256fc.png)
能否有个范例或者说明,,,X1,X2具体代表啥呀? 是代表左键?右键?  还是点1次,点2次,,,试了没反应呀,,,有没有例子呀?

**Comment by shuax** [comment]
这是鼠标侧键

**Comment by shuax** [comment]
写两行就双击了吧，我没试过。

**Reply by SoyKoa** [reply]
呵呵,我一直以为是单击,双击,,,这下明白了,,谢谢老大
